client/rpc-servers/src/middleware.rs

```diff
@@ -204,8 +204,9 @@ impl RpcMiddleware {
 	}
 
 	/// Called once the JSON-RPC request is finished and response is sent to the output buffer.
-	fn on_response(&self, _result: &str, started_at: std::time::Instant) {
+	fn on_response(&self, result: &str, started_at: std::time::Instant) {
 		log::trace!(target: "rpc_metrics", "[{}] on_response started_at={:?}", self.transport_label, started_at);
+		log::trace!(target: "extra_rpc_metrics", "[{}] result={:?}", self.transport_label, result);
```

Comment:
Not sure about the naming, can we not also just name it `rpc_metrics`?

Reply:
It's just to not interfere with the current setups. Node operators probably enable `rpc` target to get stats on rpc method call usage but likely don't care about the response payload in most cases, which can get quite big in size (and expensive to store) and is mostly useful in development environments to just debug. Separating both targets one can choose the granularity.

Reply:
you can already get this with `RUST_LOG="jsonrpsee=trace" (except the transport label) that log is super-noisy but shouldn't matter for development environments.

in addition jsonrpsee actually has separate tracing spans for each RPC method call to filter out stuff but it doesn't play well with logger in substrate, I should probably fix that as some point.